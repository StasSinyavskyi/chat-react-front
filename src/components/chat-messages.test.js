/* eslint-env jest */
import renderer from 'react-test-renderer';
import React from 'react';
import ReactDOM from 'react-dom';
import ChatMessges from './chat-messages';

jest.mock('./message', () => () => 'Message');

// if we have several test we can join them in describe
describe('<ChatMessges />', () => {
  const testParam = (
    <ChatMessges
      activeUser={{ }}
      activeChat={{ }}
      messages={{ }}
    />
  );

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(testParam, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('render correctly', () => {
    const tree = renderer
      .create(testParam)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
