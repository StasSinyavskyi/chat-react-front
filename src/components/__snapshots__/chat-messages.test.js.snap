// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatMessges /> render correctly 1`] = `
<h1
  className="MuiTypography-root-29 MuiTypography-display1-33"
>
  There is no messages yet...
</h1>
`;
